lib/Immediate/ImmediateImpl.h

```diff
@@ -34,7 +34,7 @@ namespace swift {
 
 namespace immediate {
 
-bool loadSwiftRuntime(StringRef runtimeLibPath);
+void *loadSwiftRuntime(StringRef runtimeLibPath);
```

Comment:
I know it was like this before, but mind adding a doc comment? (It's not obvious what the return value is.)